### PR TITLE
[SD-1442] Implement sliding expiry for content items

### PR DESCRIPTION
**Pull request by akintolga**
[SD-1442] Sliding Expiry
Expiry value of the content item will be reset when:
- Content is created/fetched
- Content is updated
- Content is sent to a desk/stage/workspace
- Spiked/Unspiked

[SD-1452] Updating expiry
- Expiry value of the content items in a stage will be reset when the stage expiry value is updated.

### CODE REVIEW

File: apps/archive/common.py [hunk 1]
```diff
@@ -185,6 +185,51 @@ def is_assigned_to_a_desk(doc):
     return doc.get('task') and doc['task'].get('desk')
 
 
+def get_item_expiry(app, desk, stage):
+    expiry_minutes = app.settings['CONTENT_EXPIRY_MINUTES']
```
Comment (sivakuna-aap) [inline apps/archive/common.py]: We can directly import `CONTENT_EXPIRY_MINUTES` from `settings.py` module

Comment (ioanpocol) [inline apps/archive/common.py]: :+1:

**Comment by amagdas** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**